app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -98,3 +98,7 @@ table.table-show {
 .form-control {
   --bs-border-color: #979797;
 }
+
+.form-check-input {
+  padding: 0
```

Comment:
This was necessary in order to use the `form-control` class to get the correct border on the checkbox component but not increase it's padding.

Reply:
If this change is only for the checkbox component, I wonder if adding the `p-0` class there would work OK without having side effects.

Reply:
See comment below. This was about the border, so an odd changed... I updated it.